### DISCUSSION TITLE
How to implement Filtering/Pagination in existing or new query using dynamic type schema?

### DISCUSSION THREAD

**Discussion by Sangeetha-Murugesan** [ChilliCream/hotchocolate#6368] [Q&A]
### Product

Hot Chocolate

### Is your feature request related to a problem?

This is my dynamic schema json, using this I could create an new field under existing type with arguments and response types. Along with this, I am trying to implement the filtering, pagination & sorting using fieldDescriptor.But getting the error: "No default filter convention found. Call `AddFiltering()` on the schema builder".

AddFiltering() is already registerd against the schema, do we need to add the same in TypeModule file? If so, where do we have to add ?

DynamicSchema JSON:
`{
    "name": "saleorder_ScreenLaunch_out",
    "extension": true,
    "fields": [
      {
        "name": "getCustDetailInfo", 
        "arguments": [ 
          {
            "name": "Customer",
            "type": "String"
          }
        ],
        "type": "[getCustDetailResponse]" 
      }
    ]
  }`

### The solution you'd like

```
  var fieldDescriptor = ObjectFieldDescriptor.From(context, field);                      
                        fieldDescriptor.UseOffsetPaging().UseFiltering().UseSorting();
                        fieldDescriptor.FromJson();
                        fieldDescriptor.CreateDefinition();
```
                        
This should implement the pagination,filtering, sorting with out any errors

**Comment by Sangeetha-Murugesan** [comment]
Hi , any workaround for this or is this a feature in roadmap?

**Comment by Sangeetha-Murugesan** [comment]
hi @michaelstaib , any update ?

**Reply by michaelstaib** [reply]
Can you create a minimal repro?

**Reply by SeanTAllen** [reply]
@michaelstaib I'm not sure if it is the same problem, but it appears to be. See https://github.com/ChilliCream/graphql-platform/discussions/6975. If it is the same, we have a minimal reproduction attached to it.

The symptoms look the same and we are also attempting to get filtering working with dynamic types, so it is probably the same.
